src/main/java/uk/gov/hmcts/reform/sscs/ccd/domain/HearingRoute.java

```diff
@@ -0,0 +1,26 @@
+package uk.gov.hmcts.reform.sscs.ccd.domain;
+
+import com.fasterxml.jackson.annotation.JsonEnumDefaultValue;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Getter;
+
+@Getter
+public enum HearingRoute {
```

Comment:
I don't get why the field 'type' is on this enum as jackson should be able to map to the enum name - but you've probably done it for a reason.